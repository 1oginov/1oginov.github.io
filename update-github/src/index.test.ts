/* eslint-disable @typescript-eslint/no-var-requires */

const { handler } = require('index');

it('returns status code 200', async () => {
  const response = await handler();

  expect(response.statusCode).toBe(200);
});

it('returns the same response', async () => {
  const response = await handler();

  expect(response.body).toMatchSnapshot();
});
