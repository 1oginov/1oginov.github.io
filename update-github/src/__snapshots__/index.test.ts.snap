// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns the same response 1`] = `"Hello, world!"`;
